From 5dc239a1179cd627f2bdb5cbeadce53d91bbae79 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:31:56 +0900
Subject: [PATCH 02/23] Revert "arm64: dts: r8a77965: Add MFIS Lock device
 node"

This reverts commit a29f51f0e6aa ("arm64: dts: r8a77965: Add MFIS
Lock device node")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a77965.dtsi | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965.dtsi b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
index 2255890..24bb42a 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
@@ -696,13 +696,6 @@
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 		};
 
-		mfis: mfis@e6260000 {
-			compatible = "renesas,mfis-lock-r8a77965",
-				     "renesas,mfis-lock";
-			reg = <0 0xe6260000 0 0x1000>;
-			#hwlock-cells = <1>;
-		};
-
 		pfc: pin-controller@e6060000 {
 			compatible = "renesas,pfc-r8a77965";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

